releases/CHANGELOG-v0.80.3.md

```diff
@@ -0,0 +1,14 @@
+
+<a name="v0.80.3"></a>
+# [v0.80.3] - 2025-01-27
```

Comment:
Could we add a section # Attention and call out the known chrony issue as described in the release notes. 

Example of Attention section: https://github.com/Azure/aks-engine-azurestack/pull/227/files

Reply:
Updated